develop/dev-guide-proxysql-integration.md

```diff
@@ -5,9 +5,22 @@ summary: Introduce how to integrate TiDB with ProxySQL step by step.
 
 # Integrate TiDB with ProxySQL
 
+<CustomContent platform="tidb">
+
 This document describes how to integrate **TiDB** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:
 
 - [TiDB Documentation](/overview.md)
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:
+
+- [TiDB Cloud Documentation](/tidb-cloud/tidb-cloud-intro.md)
```

Comment:
```suggestion
This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. 
```

Reply:
When users are viewing this doc, they are already opening the Cloud docs website, so there is no need to direct them to the Cloud introduction again.